src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PublicClassXmlComment/PublicClassXmlCommentFix.cs

```diff
@@ -0,0 +1,215 @@
+        {
+            cancellation.ThrowIfCancellationRequested();
+
+            var rootNode = await document
+                .GetSyntaxRootAsync(cancellation)
+                .ConfigureAwait(false);
+            if (!(rootNode?.FindNode(span) is ClassDeclarationSyntax classDeclarationSyntax))
+            {
+                return document;
+            }
+
+			var semanticModel = await document
+				.GetSemanticModelAsync(cancellation)
+				.ConfigureAwait(false);
+			if (semanticModel == null)
+			{
+				return document;
+			}
+
+			var classTypeSymbol = semanticModel.GetDeclaredSymbol(classDeclarationSyntax, cancellation);
+			if (string.IsNullOrEmpty(classTypeSymbol?.Name))
+			{
+				return document;
+			}
+
+            var newRootNode = GetRootNodeSyntaxWithDescription(rootNode, classDeclarationSyntax, classTypeSymbol.Name, option, cancellation);
+            var newDocument = document.WithSyntaxRoot(newRootNode);
+
+            return newDocument;
+        }
+
+        private SyntaxNode GetRootNodeSyntaxWithDescription(SyntaxNode rootNode, ClassDeclarationSyntax classDeclarationSyntax,
+			string className, FixOption option, CancellationToken cancellation)
+        {
+            cancellation.ThrowIfCancellationRequested();
+
+			var description = GenerateDescriptionFromCamelCase(className);
+
+            switch (option)
+            {
+                case FixOption.NoXmlComment:
+				case FixOption.NoSummaryTag:
+					return AddXmlCommentDescription(rootNode, classDeclarationSyntax, description, cancellation);
+				case FixOption.EmptySummaryTag:
+					return AddDescription(rootNode, classDeclarationSyntax, description, cancellation);
+				default:
+					return classDeclarationSyntax;
+            }
+        }
+
+		private SyntaxNode AddDescription(SyntaxNode rootNode, ClassDeclarationSyntax classDeclarationSyntax,
+			string description, CancellationToken cancellation)
+		{
+			cancellation.ThrowIfCancellationRequested();
+
+			var summaryTag = classDeclarationSyntax
+				.GetLeadingTrivia()
+				.Select(t => t.GetStructure())
+				.OfType<DocumentationCommentTriviaSyntax>()
+				.SelectMany(d => d.ChildNodes())
+				.OfType<XmlElementSyntax>()
+				.Where(n => PublicClassXmlCommentAnalyzer.XmlCommentSummaryTag.Equals(n?.StartTag?.Name?.ToString(), StringComparison.Ordinal))
+				.First();
+
+			var xmlDescription = XmlText(description);
+			var newContent = new SyntaxList<XmlNodeSyntax>(xmlDescription);
+			var newSummaryTag = summaryTag.WithContent(newContent);
+
+			return rootNode.ReplaceNode(summaryTag, newSummaryTag);
+		}
+
+		private SyntaxNode AddXmlCommentDescription(SyntaxNode rootNode, ClassDeclarationSyntax classDeclarationSyntax,
+			string description, CancellationToken cancellation)
+        {
+            cancellation.ThrowIfCancellationRequested();
+
+			var xmlDescriptionTrivia = Trivia(
+				DocumentationComment(
+					XmlSummaryElement(
```

Comment:
Most of the documentation tools as well as Visual Studio generate summary with this pattern:
```xml
<summary>
Description
</summary>
```
So, I think we also need to stick with it instead of:
```xml
<summary>Description</summary>
```

Reply:
@SENya1990 done